packages/block-library/src/image/edit.native.js

```diff
@@ -310,28 +307,20 @@ export class ImageEdit extends React.Component {
 		}
 	}
 
-	getIcon( iconType ) {
-		let iconStyle;
-		switch ( iconType ) {
-			case ICON_TYPE.RETRY:
-				return <Icon icon={ SvgIconRetry } { ...styles.iconRetry } />;
-			case ICON_TYPE.PLACEHOLDER:
-				iconStyle = this.props.getStylesFromColorScheme(
+	getPlaceholderIcon() {
+		return (
+			<Icon
+				icon={ icon }
```

Comment:
To be honest it's kinda confusing that the icon for the placeholder is called `icon`. I know it was like that before but wdyt about changing it to `placeholderIcon`?

Reply:
I agree. I changed the name of the icon 👍